Source/ServerTrustEvaluation.swift

```diff
@@ -352,9 +334,30 @@ public final class DisabledEvaluator: ServerTrustEvaluating {
     public func evaluate(_ trust: SecTrust, forHost host: String) throws { }
 }
 
-extension Bundle {
+// MARK: - Extensions
+
+public extension Array where Element == ServerTrustEvaluating {
+    #if os(Linux)
+    // Add this same convenience method for Linux.
+    #else
+    /// Evaluates the given `SecTrust` value for the given `host`.
+    ///
+    /// - Parameters:
+    ///   - trust: The `SecTrust` value to evaluate.
+    ///   - host:  The host for which to evaluate the `SecTrust` value.
+    /// - Returns: Whether or not the evaluator considers the `SecTrust` value valid for `host`.
+    func evaluate(_ trust: SecTrust, forHost host: String) throws {
+        for evaluator in self {
+            try evaluator.evaluate(trust, forHost: host)
+        }
+    }
+    #endif
+}
```

Comment:
Seems like we should probably hide this internally or inside `AlamofireExtended` as well. Pretty sure it can't be done with `AlamofireExtended` which is probably why you didn't already do it if I had to guess.

Reply:
My standard for hiding was: if we don't own either the extended or returned types, put it inside the extension point. If we do, like for this array convenience, or the `headers`, I didn't.

Reply:
Thanks for the clarification. I think that's a really sound approach.